README.md

```diff
@@ -158,6 +158,68 @@ Just want to log violations in production:
 ActionController::Parameters.action_on_invalid_parameters = :log
 ```
 
+## ControllerSupport::PermittedParameters
```

Comment:
the constant does not exist ... so copy-pasting it would break ... use a regular heading like `Controller support`